CHANGES.rst

```diff
@@ -0,0 +1,20 @@
+======================================
+CHANGES for Prometheus CrateDB Adapter
+======================================
+
+Unreleased
+==========
+
+- Use Postgres wire protocol (pgx client library) to connect to CrateDB:
+
+  - This change requires CrateDB 3.1.0 or newer!
+
+  - Connections can be configured via ``crate.yml`` configuration file using
+    the ``-config.file`` flag.
```

Comment:
Out of scope for this PR, but wouldn't it be nicer to be able to specify a connection-string directly? Would be docker friendlier as one doesn't have to bundle the right `crate.yml`

Reply:
Agree, that's a valid point. Please make an issue with a feature request.